includes/discovery/discovery-protocols.inc.php

```diff
@@ -204,14 +216,42 @@
         $IndexId = key($lldp['lldpNeighborPortIndexId']);
 
         $local_ifName = $lldp['lldpNeighborPortId'][$IndexId];
-        $local_port_id = find_port_id('gigabitEthernet ' . $local_ifName, null, $device['device_id']);
 
+        //try to find our local port
+        $local_port_id = find_port_id('gigabitEthernet ' . $local_ifName, null, $device['device_id']); //normal port name
+        if (! $local_port_id) {
```

Comment:
surround with jetstream check

Reply:
> surround with jetstream check

sorry, please explain. i will rewrite, but i don't understand what ?

Reply:
Your :fiber and :copper checks are only valid for jestream.  Wrap them in a check ;)

Reply:
Ok, two new function was added in 
includes/discovery/functions.inc.php

hope this will be OK